site/content/guides/claude-code-monitoring.md

```diff
@@ -0,0 +1,303 @@
+});
+```
+
+Claude Code will confirm before creating the check and spec files with the configuration and script, respectively.
+
+You can prompt for multiple checks at the same time, as the output is relatively token-efficient. As you can imagine, this unlocks the creation of a large number of synthetic monitors relatively quickly. Another use case that may be useful for larger projects, try prompting Claude Code with something like:
+
+
+> update every `.check.ts` file for a check that touches [danube-web.shop](http://danube-web.shop/), and change
+> the frequency to every 30 minutes.
+> 
+
+Not only did this request work, but I even got sensible feedback about the checks that currently had no frequency setting:
+
+```bash
+  The url.check.ts and api.check.ts files don't have individual frequency settings -
+  they inherit the default frequency from the project configuration in
+  checkly.config.ts.
+```
+
+Once we’re done creating new checks, it’s time to run them all with:
+
+```bash
+npx checkly test
+```
+
+When I ran a number of generated tests, the only issue I ran into was a failure with the message: `Error: expect.toBeVisible: Error: strict mode violation: getByText('Crime & Thrillers') resolved to 2 elements:`. As the error message implies, the issue is that this locator:
+
+```ts
+  await expect(page.getByText('Fantasy')).toBeVisible();
+```
+
+Found multiple results on the page. There are a number of ways to fix this with Claude Code, we could prompt Claude Code with:
+
+> revise locators in item-visibility.spec.ts to look at only the first result
+> 
+
+This would change every locator, which may or may not be what you want! You could also paste in the error message to Claude and ask it to fix the script. Whatever route you take the result will be an edit to:
+
+```ts
+  await expect(page.getByText('Fantasy').first()).toBeVisible();
+```
+
+If you’re having trouble reading the generated tests, take a look at [Checkly’s Playwright documentation](https://www.checklyhq.com/learn/playwright/). Once Claude Code added a single `.first()` to my check, everything was passing:
+
+![Feedback on the terminal](/guides/images/claude-monitoring-04.png)
+
+## Deploy With the Checkly CLI
+
+In our example, we want to create several checks at once, so it’s worth checking whether the configuration is properly formatted before deployment. Many problems would have been revealed while running `test` but if, for example, the [logical IDs](https://www.checklyhq.com/docs/cli/constructs-reference/#synthetic-checks) of current and existing checks are colliding, we’ll need to run `deploy` to detect that. Run the `deploy` command with the `-p` preview flag:
+
+```bash
+npx checkly deploy -p
+```
+
+This command won’t deploy anything, just parse our project and show us what will be created when we run the command without the preview flag.
+
+![Feedback on the terminal](/guides/images/claude-monitoring-05.png)
```

Comment:
Nit: this screenshot is slightly confusing. It shows 7 checks. This entire guide only discusses two.